### DISCUSSION TITLE
Please release the source code of gatsbyjs.com

### DISCUSSION THREAD

**Discussion by Nowaker** [gatsbyjs/gatsby#34452] [Ideas / Feature Requests]
https://www.gatsbyjs.com/ is generated by Gatsby. Please release the source of this website as open source so we have a great example of a **full & real** website.

Gatsby starters may be useful, but they aren't as useful as a full site that has many aspects covered - especially SEO. We need real world examples and the source of gatsbyjs.com would qualify.

**Comment by LekoArts** [comment]
You could go to https://www.gatsbyjs.com/showcase/ and filter by "Open Source"
